omod/src/main/webapp/pages/conditionlist/addCondition.gsp

```diff
@@ -46,11 +46,13 @@
     <div class="horizontal">
         <ul id="concept-and-date">
             <li class="group">
+                <h3 class="inline">${ui.message('coreapps.conditionui.condition')} </h3>
```

Comment:
Is this addition part of removing history of?

Reply:
No, This is to address the issue of labeling the fields. As it was a very minor change I included it in this PR.

Reply:
Then that should be a separate ticket and hence pull request.